packages/office-ui-fabric-react/src/utilities/contextMenu/contextMenuUtility.ts

```diff
@@ -0,0 +1,30 @@
+import { IContextualMenuItem } from '../..';
+
+/**
+ * Determines the effective checked state of a menu item.
+ *
+ * @param item {IContextualMenuItem} to get the check state of.
```

Comment:
hmm minor nit, shouldn't this folder and file be `contextualmenu` rather than `contextmenu`? Consistency is good.